Plugins (#3438)
ElektraInitiative/libelektra: doc/news/_preparation_next_release.md

## Plugins

> We updated following [plugins](https://www.libelektra.org/plugins/readme):
> 
> We do not maintain following plugins anymore (status "maintained" was removed):

Reviewer: ```suggestion
We updated the `maintained` status of the following [plugins](https://www.libelektra.org/plugins/readme):
```
